include/volume/volume_sequence_of_balls.hpp

```diff
@@ -30,7 +30,9 @@
 #include "convex_bodies/ballintersectconvex.h"
 #include "convex_bodies/zonoIntersecthpoly.h"
 #include "convex_bodies/vpolyintersectvpoly.h"
-#include "volume/rounding.hpp"
+#include "preprocess/mee_rounding.hpp"
```

Comment:
same here

Reply:
no, only in `volume_sequence_of_hpoly.hpp`

Reply:
ok so those includes are not needed

Reply:
Yes of course, removed.